### PR TITLE
South Africa - Hilton Otto

### PR DESCRIPTION
**Pull request by theTrainMan**
Made some amendments and updated question8

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

**Comment by Ntshembo-Hlongwane1** [comment]
Hey Hilton, here is something that I found from you answers that you could revisit. I will have **your answer in a block quote and my in plain text** 

# Question 7: What is JavaScript:

> JAvascript is a scripting or programming language that allows you to implement complex features on web pages.

So if you had left it at **JavaScript is a scripting programming language technically you would be right**. But then what you have to know and understand is that JavaScript is more that just for making **"complex features on web pages"**, JavaScript over the past few years evolved  so much ever since **2009** **when NodeJS** was introduced allowing people to use JavaScript to creating **CLI's and Web Services**, then came **React Native 2015** and now we can create **Mobile Applications** with JavaScript finally came **ElectronJS 2019** now we can create **Desktop application for Windows, Mac and Linux**... So I hope you see that your answer might need some amending due to you **including what JavaScipt does which in all honest is a seperate question on it's own than this one?**

File: homework.md [hunk 1]
```diff
@@ -44,37 +44,39 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
-A programming language used on the internet
+JAvascript is a scripting or programming language that allows you to implement complex features on web pages.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
 When people want to show off code to each other they can put it on Github
+It is a collaboration tool with features such as wikis and task management for projects
+It also provides access control.
```
Comment (Jontas77) [inline homework.md:56]: Hi Hilton where is the third one?

**Comment by Jontas77** [comment]
Hi Hilton I don't know whether you missed question 15 for there is no change made and that answer is wrong. Please can you check it.

**Comment by Jontas77** [comment]
Hi Hilton I am done with my review.

**Pull request closed without merging**